torch_audiomentations/augmentations/background_noise.py

```diff
@@ -0,0 +1,53 @@
+import random
+import torch
+from ..core.transforms_interface import BasicTransform
+from ..utils.file import find_audio_files, load_audio
+from ..utils.dsp import calculate_rms, calculate_desired_noise_rms
+
+
+class ApplyBackgroundNoise(BasicTransform):
+    """
+    Applies a background noise to the input signal.
+    """
+
+    def __init__(self, bg_path, snr_in_db, p=0.5):
```

Comment:
Could we support a distribution on `snr_in_db` instead of a fixed number? That would be more useful IMO.